src/test/java/org/opensearch/bootstrap/JarHell.java

```diff
@@ -0,0 +1,51 @@
+/*
```

Comment:
Are these the correct license headers?

Reply:
hmm..they aren't.. fixed them

Reply:
Note; I don't see a change in the headers yet

Reply:
I still don't see any update in these headers, @DarshitChanpura please update

Reply:
done